features/application-mgt/org.wso2.carbon.identity.application.mgt.server.feature/resources/identity/authenticationTemplates/session.json

```diff
@@ -0,0 +1,51 @@
+{
+  "category": "user_based",
+  "name": "Concurrent-Session-Management",
+  "title": "Concurrent Session Management Template",
+  "summary": "Prompts session handling to the users who belongs to any of the given set of roles based on currently active session count.",
+  "preRequisites": [
+    "Change the rolesList parameter to an array of roles of which users need to prompt for session handling.",
+    "Modify the maxSessionCount parameter as required."
+  ],
+  "parametersDescription": {
+    "rolesToStepUp": "An array of roles of which users should be prompted for session handling.",
+    "maxSessionCount": "Maximum number of allowed concurrent sessions for the role which a particular user belongs to."
+  },
+  "defaultStepsDescription": {
+    "Step 1": "Basic (Password) authenticator.",
+  },
+  "authenticationSteps":1,
+  "defaultAuthenticators": {
+    "1": {
+      "local": [
+        "BasicAuthenticator"
+      ],
+      "federated": []
+    }
+  },
+  "helpLink": "https://docs.wso2.com/display/IS570/Configuring+Role-Based+Adaptive+Authentication",
```

Comment:
can't we link to 5.9.0

Reply:
Once the doc for the new functions is added.
We can directly link to that.